eng/jacoco-test-coverage/pom.xml

```diff
@@ -144,7 +144,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-data-appconfiguration</artifactId>
-      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
+      <version>1.1.6</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
```

Comment:
Might need the revApi fix.

Reply:
Yep. the fix in TA+FR release PR. Trying to merge https://github.com/Azure/azure-sdk-for-java/pull/15963 first, so the problem will be removed.